helpers/password.go

```diff
@@ -0,0 +1,8 @@
+package helpers
+
+import "golang.org/x/crypto/bcrypt"
+
+func HashPassword(password string) (string, error) {
+	bytes, err := bcrypt.GenerateFromPassword([]byte(password), 14)
```

Comment:
@junan Is there any reason for using `14` here? I saw that `bcrypt` provided some constants 🤔 
```
const (
	MinCost     int = 4  // the minimum allowable cost as passed in to GenerateFromPassword
	MaxCost     int = 31 // the maximum allowable cost as passed in to GenerateFromPassword
	DefaultCost int = 10 // the cost that will actually be set if a cost below MinCost is passed into 
)
```

Reply:
@carryall Increasing the cost factor will slow the hash function and it is good  security practice to slow the hash function unless it effects in the UX(for 14 cost factor, it takes less than 1 second on my machine to generate the password, so it does not affecting). For example, if the database is hacked, then the hacker/attacker will use brute-force method to know the password. But if we used a higher cost factor, it will make extremely difficult to crack the password then using lower cost factor. This this article(https://labs.clio.com/bcrypt-cost-factor-4ca0a9b03966), it said that default cost factor 10 is a bit weaker, it was 4 years ago and now hardware gets more powerful. So its better to use higher cost factor when possible(unless it is an an issue for registration/login)